backend/corpora/lambdas/api/v1/curation/collection_uuid/__init__.py

```diff
@@ -0,0 +1,26 @@
+from flask import g
```

Comment:
Stepping back a bit, I'm not sure I understand why the file tree is organized as it is. For starters, `../collection/` and `../collection_uuid/`...? What's the difference, exactly?

And why are we putting this logic in this `__init__.py` file?

Reply:
We don't seem to be using this as common code within the other files in this module

Reply:
I'm not seeing the lines within which anything is being organized (not in this PR specifically, but generally)

`collection_uuid/` directory holds `revision.py` and `dataset.py` but the collection deletion code is going in the init file for that dir, and then there is a whole separate `collection/` dir?

Reply:
your right I put some of the files in the wrong place. In general, the goal is the have the API handlers be in roughly the same location as the API path. For example: `Delete /curation/v1/collection/{collection_uuid}` handler should live in `/curation/v1/collection/collection_uuid.py`, `/curation/v1/collection/collection_uuid/__init__.py`, or `/curation/v1/collection/collection_uuid/delete.py`. This whole API directory needs reorganizing, but I don't plan on doing it in this PR.

Reply:
> This whole API directory needs reorganizing, but I don't plan on doing it in this PR.

Agreed, we can leave to a later date.